### PR TITLE
Rcint 6558

### PR DESCRIPTION
**Pull request by huanhulan**

### CODE REVIEW

**Comment by RingCentralIntegrationDev** [comment]
Something went wrong. Investigate!

**Comment by RingCentralIntegrationDev** [comment]
Something went wrong. Investigate!

**Comment by RingCentralIntegrationDev** [comment]
Something went wrong. Investigate!

**Comment by RingCentralIntegrationDev** [comment]
Something went wrong. Investigate!

**Comment by RingCentralIntegrationDev** [comment]
Something went wrong. Investigate!

File: src/modules/Conversation/index.js [hunk 1]
```diff
@@ -298,7 +333,21 @@ export default class Conversation extends RcModule {
     return this.state.recipients;
   }
 
-  get messageStoreUpdatedAt() {
-    return this.state.messageStoreUpdatedAt;
-  }
+  @getter
+  messageTexts = createSelector(
```
Comment (embbnux) [inline src/modules/Conversation/index.js]: get messageTexts

**Comment by RingCentralIntegrationDev** [comment]
Something went wrong. Investigate!

**Pull request merged**